hi-ml-histopathology/src/histopathology/utils/report_utils.py

```diff
@@ -15,8 +15,51 @@
                                 get_tags_from_hyperdrive_run)
 
 
+def run_has_val_and_test_outputs(run: Run) -> bool:
+    """Checks whether the given run has both validation and test outputs files.
+
+    :param parent_run: The run whose outputs to check.
+    :raises ValueError: If the run does not have the expected output file(s).
+    :return: `True` if the run has validation and test outputs, `False` if it is a legacy run with
+        only test outputs.
+    """
+    outputs_filename = "test_output.csv"
+    val_outputs_filename = "val/" + outputs_filename
+    test_outputs_filename = "test/" + outputs_filename
+
+    outputs_folder = "outputs/"
+    all_available_files: List[str] = run.get_file_names()
+    output_files = [filename[len(outputs_folder):] for filename in all_available_files
+                    if filename.startswith(outputs_folder)]
+    if val_outputs_filename in output_files and test_outputs_filename in output_files:
+        return True
+    elif outputs_filename in output_files:
+        return False
+    else:
+        raise ValueError(f"Run {run.display_name} ({run.id}) does not have the expected files: {output_files}")
```

Comment:
I think this error message may be a bit confusing since it doesn't tell the user the expected file it is looking for (`outputs_filename` )